packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -68,6 +68,12 @@ export interface PipelineProps {
    */
   readonly artifactBucket?: s3.IBucket;
 
+  /**
+   * The IAM role to be assumed by this Pipeline.
+   * If not specified, a new IAM role will be created.
```

Comment:
Use a `@default` marker in the docs to indicate the default behavior.

Reply:
@rix0rrr do you have an example of the `@default` format that's expected? I'm not familiar with it, and I can't see where this is used anywhere else in this file

Reply:
Here is an example:

https://github.com/awslabs/aws-cdk/blob/master/packages/@aws-cdk/aws-lambda/lib/function.ts#L115

I would not copy the second line "Both supplied and generated...". That feature is a pattern all over the library that can be assumed and doesn't have to be documented in every location.

Reply:
Gotcha, done that. How does it look?